File: Crossplatform/c_sharp_version/ServerApp/domain/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    public class Employee
    {
        private string id;
        private string id_agency;
        private string login_user;
        private string login_pswd;

        public Employee(string id, string id_agency, string user, string pswd)
        {
            this.id = id;
            this.id_agency = id_agency;
            this.login_user = user;
            this.login_pswd = pswd;
        }

        public string getEmployeeId()
        {
            return id;
        }
        public void setEmployeeId(string id)
        {
            this.id = id;
        }

        public string getAgencyId()
        {
            return id_agency;
        }
        public void setAgencyId(string id)
        {
            this.id_agency = id;
        }

        public string getLoginUser()
        {
            return login_user;
        }
        public void setLoginUser(string user)
        {
            this.login_user = user;
        }

        public string getLoginPswd()
        {
            return this.login_pswd;
        }

        public void setLoginPswd(string pswd)
        {
            this.login_pswd = pswd;
        }

        public override string ToString()
        {
            return id + " "  +id_agency+ " " + login_user + '\n';
        }
    }
}

File: Hibernatfe Java HW/ServerApp/src/main/java/domain/Employee.java
package domain;


public class Employee {
    private String id; // direct id
    private String id_agency;
    private String login_user;
    private String login_pswd;
    public Employee(String employee_id, String id_agency, String login_user, String login_pswd) {

        this.id = employee_id;
        this.id_agency = id_agency;
        this.login_user = login_user;
        this.login_pswd = login_pswd;
    }

    public String getId_agency() {
        return id_agency;
    }

    public void setId_agency(String id_agency) {
        this.id_agency = id_agency;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getLogin_user() {
        return login_user;
    }

    public void setLogin_user(String login_user) {
        this.login_user = login_user;
    }

    public String getLogin_pswd() {
        return login_pswd;
    }

    public void setLogin_pswd(String login_pswd) {
        this.login_pswd = login_pswd;
    }

    @Override
    public String toString() {
        return  id + " " + login_user + " " + id_agency + '\n';
    }
}
